optuna/samplers/_qmc.py

```diff
@@ -152,7 +152,7 @@ def __init__(
         warn_independent_sampling: bool = True,
     ) -> None:
         self._scramble = scramble
-        self._seed = seed or np.random.PCG64().random_raw()
+        self._seed = np.random.PCG64().random_raw() if seed is None else seed
```

Comment:
The change looks solid to me.
How about adding a test case to detect the bug in the corresponding test case?

The confirmed that `test_sample_relative_seeding` failed if I added the case `seed=0`.
```python
@pytest.mark.parametrize("seed", [0, 12345])
@pytest.mark.parametrize("scramble", [True, False])
@pytest.mark.parametrize("qmc_type", ["sobol", "halton"])
def test_sample_relative_seeding(seed: int, scramble: bool, qmc_type: str) -> None:
```

Reply:
Thanks!